### PR TITLE
Task dom js

### PR DESCRIPTION
**Pull request by SamVal007**
### JS-DOM

[demo](https://samval007.github.io/kottans-frontend/Task_DOM_SnookerPlayers/)
[codebase](https://github.com/SamVal007/kottans-frontend/tree/main/Task_DOM_SnookerPlayers)

Sorry, I understand my mistakes, i hope:)
Please, review.

### CODE REVIEW

File: submissions/SamVal007/task-js-dom/dom.js [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+    p.textContent = item;
+    div.appendChild(p);
+  });
+
+
+  fragment.appendChild(div);
+  return fragment.firstElementChild.innerHTML;
+}
+
+const displayContent = (activeElem) => {
```
Comment (ustymchyk) [inline submissions/SamVal007/task-js-dom/dom.js:125]: It's not needed to pass the whole Element object here as an argument.
There are two ways: destructuring or pass only id.

File: submissions/SamVal007/task-js-dom/dom.js [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+let currentCarIndex = "0";
+
+const toggleActiveLink = (newActiveEl) => {
+  document.querySelector('.active').classList.toggle('active');
+  newActiveEl.classList.toggle('active');
+}
+
+const generateCarDescription = (id) => {
+  const fragment = document.createDocumentFragment();
+  const div = document.createElement('div');
```
Comment (ustymchyk) [inline submissions/SamVal007/task-js-dom/dom.js:111]: This variable name doesn't make any sense. I think it should be renamed.

File: submissions/SamVal007/task-js-dom/dom.js [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+      "In June 2016, he became ambassador of electronic snooker simulator app Snooker Live Pro. He was an avid gamer but gave up the hobby in April 2017, believing he was spending too much time playing games and it was affecting his snooker form."
+    ]
+  },
+]
+
+const playerPhoto = document.querySelector('.photo');
+const playerName = document.querySelector('.player-name');
+const playerborn = document.querySelector('.born');
+const playerCurrentRanking = document.querySelector('.CurrentRanking');
+const playerCareerWinnings = document.querySelector('.CareerWinnings');
```
Comment (ustymchyk) [inline submissions/SamVal007/task-js-dom/dom.js:98]: Kebab-case is more common for html classes(e.g. `<p class="current-ranking"></p>`)

File: submissions/SamVal007/task-js-dom/dom.js [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+  playerPhoto.setAttribute('src', PLAYERS[activeElem.id].pic);
+  playerName.textContent = PLAYERS[activeElem.id].name;
+  playerborn.textContent = PLAYERS[activeElem.id].born;
+  playerCurrentRanking.textContent = PLAYERS[activeElem.id].CurrentRanking;
+  playerCareerWinnings.textContent = PLAYERS[activeElem.id].CareerWinnings;
+}
+
+const handler = ({
+  target
+}) => {
```
Comment (ustymchyk) [inline submissions/SamVal007/task-js-dom/dom.js:135]:
```suggestion
const handler = ({ target  }) => {
```
Maybe it will be more readable for you.

**Review by ustymchyk — COMMENTED** [review]
I've reviewed only .js file with common mistakes.
All comments are only recommendations.

File: submissions/SamVal007/task-js-dom/index.html [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+          require a rest or spider. When he first displayed this left-handed ability in the 1996 World Championship
+          against Alain Robidoux, the Canadian accused him of disrespect and refused to shake hands after the match.
+        </p>
+      </div>
+    </div>
+  </main>
+  <script src="./dom.js"></script>
+</body>
+
+</html>
```
[the remark this answers is not in the document]
Comment (SamVal007) [reply submissions/SamVal007/task-js-dom/index.html]: Yep! Thank you!:)

**Comment by SamVal007** [comment]
> I've reviewed only .js file with common mistakes.
> All comments are only recommendations.

Thank you A lot!

File: submissions/SamVal007/task-js-dom/dom.js (hunk 1 above)
Comment (SamVal007) [reply submissions/SamVal007/task-js-dom/dom.js:125]: Yep! Also thought about this!!
Big thanks!

**Pull request closed without merging**